learning.py

```diff
@@ -908,6 +908,16 @@ def score(learner, size):
     return [(size, mean([score(learner, size) for t in range(trials)]))
             for size in sizes]
 
+
+def grade_learner(predict, tests):
```

Comment:
I think this would be better if it returned the ratio (i..e. percent) of correct tests, rather than the total.
Then we can call it with `grade_learner(predicter, tests) > 2/3`, regardless of length of tests.

Also, `for t in tests` should be `for (X, y) in tests`
Or the whole thing could be `return mean(int(predict(X) == y) for X, y in tests)`